### DISCUSSION TITLE
How install PiVPN on custom subnet?

### DISCUSSION THREAD

**Discussion by 4k3or3et** [pivpn/pivpn#1851] [Q&A]
Hi All,

i need to install PiVPN to operate on 10.8.0.0/24 network. As far as I can see, the script randomly assigned network rage during installation. Which part of the script I should change to set custom network IP range?

**Comment by orazioedoardo** [comment]
You can do the unattended install to set a custom range https://docs.pivpn.io/guides/user-data/
